gnomad_qc/v4/assessment/calculate_per_sample_stats.py

```diff
@@ -523,6 +603,16 @@ def main(args):
         choices=["exomes", "genomes"],
         help="Data type (exomes or genomes) to produce summary stats for.",
     )
+    parser.add_argument(
+        "--autosomes-only",
+        help="Whether to restrict analysis to autosomes only.",
```

Comment:
This argument only applies to the part you want to change 
the high ab het non-ref on autosomes, right? I'm not sure how it would be useful to filter release_ht.